API Blueprint Specification.md

```diff
@@ -767,6 +776,16 @@ Defined by the `Request` keyword followed by an optional [identifier](#def-ident
 
     + Request <identifier> (<Media Type>)
 
+This section **should** include at least one of the following nested sections:
+
+- [`0-1` Headers section](#def-headers-section)
+- [`0-1` Attributes section](#def-attributes-section)
+- [`0-1` Body section](#def-body-section)
+- [`0-1` Schema section](#def-schema-section)
+- [`0-1` URI parameters section](#def-uriparameters-section)
+
+The URI parameters section included in this section describes the URI parameters attached to the request. They can be used to represent different responses based on different parameter values sent along with a request.
```

Comment:
I'd like to see a note mentioning that when you have two URI Parameter sections (request/action and a request level URI Parameter section) one overrides the other and there is no inheritance.

I think this might be ambiguous on how this is handled otherwise.